ExponentialRamsey/Prereq/Constructive.lean

```diff
@@ -106,13 +106,12 @@ theorem isRamseyValid_myOtherLabelling'_zero {α : Type*} [Fintype α] [Decidabl
     by_contra!
     have := hm hab hab' this
     rw [myOtherLabelling', TopEdgeLabelling.mk_get] at this
-    simp only[eq_self_iff_true, true_or, if_true, Fin.one_eq_zero_iff,
-      Nat.succ_succ_ne_one] at this
+    simp only [true_or, if_true, Fin.one_eq_zero_iff, Nat.succ_succ_ne_one] at this
   have : ∀ i ∈ (Finset.univ : Finset α), (Finset.filter (fun x : _ × Fin l => x.fst = i) m).card = 1 :=
     by
     rw [← Finset.sum_eq_sum_iff_of_le this, ← Finset.card_eq_sum_ones, Finset.card_univ,
         ← Finset.card_eq_sum_card_fiberwise, ← hm']
-    simp only [Finset.mem_univ, imp_true_iff]
+    intro x xm; simp only [Finset.coe_univ, Set.mem_univ]
```

Comment:
```suggestion
    simp +contextual only [Finset.coe_univ, Set.mem_univ]
```
does this work?

Reply:
ooo nice, i know a lot of places in my own code where this trick will come in handy!